src/99-appendices/11-qmri.md

```diff
@@ -62,14 +62,14 @@ Below is an example file collection for RF transmit field map `TB1EPI`:
 ```text
 └── sub-01/
      └── fmap/
-         ├── sub-01_fa-1_echo-1_TB1EPI.nii.gz
-         ├── sub-01_fa-1_echo-1_TB1EPI.json
-         ├── sub-01_fa-1_echo-2_TB1EPI.nii.gz
-         ├── sub-01_fa-1_echo-2_TB1EPI.json
-         ├── sub-01_fa-2_echo-1_TB1EPI.nii.gz
-         ├── sub-01_fa-2_echo-1_TB1EPI.json
-         ├── sub-01_fa-2_echo-2_TB1EPI.nii.gz
-         └── sub-01_fa-2_echo-2_TB1EPI.json
+         ├── sub-01_flip-1_echo-1_TB1EPI.nii.gz
```

Comment:
`flip` needs to come after `echo`, I think.

Reply:
oh but we can just render this with code if you want

Reply:
🤦  sorry I missed that, you are right.

Reply:
Done.